nixos/modules/programs/_1password-gui.nix

```diff
@@ -30,31 +30,32 @@ in
         '';
       };
 
-      package = lib.mkPackageOption pkgs "1Password GUI" {
-        default = [ "_1password-gui" ];
-      };
+      package =
+        lib.mkPackageOption pkgs "1Password GUI" {
+          default = [ "_1password-gui" ];
+        }
+        // {
+          apply =
+            pkg:
+            pkg.override {
+              polkitPolicyOwners = cfg.polkitPolicyOwners;
```

Comment:
nit:
```suggestion
              inherit (cfg) polkitPolicyOwners;
```

Reply:
I was considering this, but it was done like this originally, likely to not have the name of the module option be reliant on the package option.